### PR TITLE
mediaconch 17.01

### PR DESCRIPTION
**Pull request by g-maxime**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/mediaconch.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Mediaconch < Formula
   desc "Conformance checker and technical metadata reporter"
   homepage "https://mediaarea.net/MediaConch"
-  url "https://mediaarea.net/download/binary/mediaconch/16.12/MediaConch_CLI_16.12_GNU_FromSource.tar.bz2"
-  version "16.12"
-  sha256 "4da0bf62e0cfc0ac33d8172595af40c4dc53256ac37a291c3910ad3b80503210"
+  url "https://mediaarea.net/download/binary/mediaconch/17.01/MediaConch_CLI_17.01_GNU_FromSource.tar.bz2"
+  version "17.01"
+  sha256 "1fc3ae71e504db09311e01b63f46620302fcd1d83b56a818d5974397aac5bd27"
   revision 1
```
Comment (retokromer) [inline Formula/mediaconch.rb]: Should the revision not be removed at the new version?

Comment (JCount) [inline Formula/mediaconch.rb]: It should be.

Comment (JCount) [inline Formula/mediaconch.rb]: It should be. Once it is removed this should be good to go.

Comment (g-maxime) [inline Formula/mediaconch.rb]: Done, thanks.

**Comment by nijikon** [comment]
You are awesome, thanks for your contribution! :tada:

**Pull request closed without merging**